class/defaults.yml

```diff
@@ -29,6 +29,7 @@ parameters:
       openshift: openshift-*
       projectsyn: syn-*
       appuio: appuio-*
+      cilium: cilium*
```

Comment:
Is the namespace just `cilium`? Or are there more namespaces e.g. `cilium-system`?

Reply:
For now, it is just `cilium`. I prefer to have that whole prefix blocked for two reasons:

* Not confuse admins if there is a `cilium` and a `cilium-something-elese` namespace
* Allow the tool to branch out to multiple namespaces

Reply:
Fair enough